src/argilla/client/feedback.py

```diff
@@ -484,7 +474,7 @@ def add_records(
             ...     ]
             ... )
             >>> dataset.records
-            [FeedbackRecord(fields={"text": "This is the first record", "label": "positive"}, responses=[{"user_id": None, "values": {"question-1": {"value": "This is the first answer"}, "question-2": {"value": 5}}}])]
+            [FeedbackRecord(fields={"text": "This is the first record", "label": "positive"}, responses=[ResponseSchema(user_id=None, values={"question-1": ValueSchema(value="This is the first answer"), "question-2": ValueSchema(value=5)})])]
```

Comment:
same change here, since the `user_id` is an optional value and we're setting it to `None` in the example, makes sense to remove it from the response initialization.

Reply:
So the thing is that since it's optional it's set to `None` by default, so when we print the `FeedbackRecord` we see the `responses` as a `ResponseSchema` with all its values, so I think it's nice that we keep it as-is in the docstrings, as that's what you get when you `print(self.records)`, and also to give the users visibility about the `user_id` field in `ResponseSchema`